### ISSUE TITLE
Phaser Burst Inductive Drum

### ISSUE THREAD

**Issue by 456Xander** [simeonradivoev/MatterOverdrive#417]
When I put a burst inductive drum on by Phaser and Fire with kill Mode, it makes an Explosion every Tick where it hits.

**Comment by DaMachinator** [comment]
I believe that is intended behavior. That is the yellow striped lens, correct?

**Comment by 456Xander** [comment]
Yes its the yellow striped lens, but i dont think that it should make
Explosions every tick, cause it would be too powerful.

**Comment by DaMachinator** [comment]
The higher up you turn the damage, the bigger the explosions are. In addition, the lens significantly increases the RF cost.

From creative testing, I do not remember the explosions being very powerful either.

**Comment by ajthemacboy** [comment]
For me, when I added the drum to my phaser, the explosion never worked... it fizzes and fizzes particles but nothing ever happens... am I doing something wrong?

**Comment by simeonradivoev** [comment]
Explosions should have a delay between them. I will have to look into why it's not working properly.

**Comment by ajthemacboy** [comment]
I'm pretty sure most of us would appreciate you pushing these fixes to the 1.7 versions too if it's not too much work for you. I think it's going to be a while before 1.8 packs take off, judging by how it's been 2 years since it was released and there are still very few packs.

**Comment by simeonradivoev** [comment]
I might add small fixes, but nothing major. And there probably won't be packs for 1.8 but most likely for 1.9

**Comment by Alexismachina** [comment]
Add label here Shadow ?

**Comment by DaMachinator** [comment]
@ajthemacboy Hold shift-leftclick until the bar on the top goes to red.
